vme/src/handler.cpp

```diff
@@ -669,14 +669,18 @@ zone_type *unit_zone(const unit_data *unit)
             //      assert(IS_ROOM(unit));
             if (!unit->isRoom())
             {
-                slog(LOG_ALL, 0, "ZONE: FATAL(1): %s@%s IN NO ROOMS WHILE NOT A ROOM!!", UNIT_FI_NAME(org), UNIT_FI_ZONENAME(org));
+                slog(LOG_ALL,
+                     0,
+                     "ZONE: FATAL(1): %s@%s IN NO ROOMS WHILE NOT A ROOM!!",
+                     org->getFileIndexName(),
+                     org->getFileIndexZoneName());
```

Comment:
Another nice enhancement would be a unit->getFileIndexSymName() which would return "%s@%s", getFileIndexName(), getFileIndexZoneNameI()